target/linux/ipq40xx/files-4.14/arch/arm/boot/dts/qcom-ipq4019-qxwlan-e2600ac.dtsi

```diff
@@ -0,0 +1,265 @@
+			pinctrl-names = "default";
+			status = "okay";
+		};
+
+		serial@78b0000 {
+			pinctrl-0 = <&serial_1_pins>;
+			pinctrl-names = "default";
+			status = "okay";
+		};
+
+		i2c@78b7000 { /* BLSP1 QUP2 */
+			pinctrl-0 = <&i2c_0_pins>;
+			pinctrl-names = "default";
+
+			status = "okay";
+		};
+
+		usb3: usb3@8af8800 {
+			status = "okay";
+
+			dwc3@8a00000 {
+				#address-cells = <1>;
+				#size-cells = <0>;
+
+				usb3_port1: port@1 {
+					reg = <1>;
+					#trigger-source-cells = <0>;
+				};
+
+				usb3_port2: port@2 {
+					reg = <2>;
+					#trigger-source-cells = <0>;
+				};
+			};
+		};
+
+		crypto@8e3a000 {
+			status = "okay";
+		};
+
+		watchdog@b017000 {
+			status = "okay";
+		};
+
+		ess-switch@c000000 {
+			status = "okay";
+		};
+
+		edma@c080000 {
+			status = "okay";
+		};
+
+		leds {
+			compatible = "gpio-leds";
+
+			led1 {
+				label = "e2600ac:green:wlan0";
+				gpios = <&tlmm 50 GPIO_ACTIVE_LOW>;
+			};
+
+			led2 {
+				label = "e2600ac:green:wlan1";
+				gpios = <&tlmm 36 GPIO_ACTIVE_LOW>;
+			};
+
+			led3 {
+				label = "e2600ac:green:usb";
+				gpios = <&tlmm 32 GPIO_ACTIVE_LOW>;
+				trigger-sources = <&usb2_port1>, <&usb3_port1>, <&usb3_port2>;
+				linux,default-trigger = "usbport";
+			};
+
+			led4 {
+				label = "e2600ac:green:ctrl1";
+				gpios = <&tlmm 51 GPIO_ACTIVE_LOW>;
+			};
+
+			led5 {
+				label = "e2600ac:green:ctrl2";
+				gpios = <&tlmm 30 GPIO_ACTIVE_LOW>;
+			};
+
+			led6 {
+				label = "e2600ac:green:ctrl3";
+				gpios = <&tlmm 31 GPIO_ACTIVE_LOW>;
+			};
+		};
+
+		keys {
+			compatible = "gpio-keys";
+
+			reset {
+				label = "reset";
+				gpios = <&tlmm 18 GPIO_ACTIVE_LOW>;
+				linux,code = <KEY_RESTART>;
+			};
+		};
+	};
+};
+
+&blsp_dma {
+	status = "okay";
+};
+
+&cryptobam {
+	status = "okay";
+};
+
+&qpic_bam {
+	status = "okay";
+};
+
+&tlmm {
+	i2c_0_pins: i2c-0-pinmux {
+		pins = "gpio20", "gpio21";
+		function = "blsp_i2c0";
+		bias-disable;
+	};
+
+	mdio_pins: mdio_pinmux {
+		mux_1 {
+			pins = "gpio6";
+			function = "mdio";
+		};
+		mux_2 {
+			pins = "gpio7";
+			function = "mdc";
+		};
+
+		pinconf {
+			pins = "gpio7", "gpio6";
+			bias-pull-up;
+		};
+	};
+
+	serial_0_pins: serial0-pinmux {
+		pins = "gpio16", "gpio17";
+		function = "blsp_uart0";
+		bias-disable;
+	};
+
+	serial_1_pins: serial1_pinmux {
+		mux {
+			pins = "gpio8", "gpio9";
+			function = "blsp_uart1";
+			bias-disable;
+		};
+	};
+
+	spi_0_pins: spi_0_pinmux {
+		pinmux {
+			function = "blsp_spi0";
+			pins = "gpio13", "gpio14", "gpio15";
+		};
+		pinmux_cs {
+			function = "gpio";
+			pins = "gpio12";
+		};
+		pinconf {
+			pins = "gpio13", "gpio14", "gpio15";
+			drive-strength = <12>;
+			bias-disable;
+		};
+		pinconf_cs {
+			pins = "gpio12";
+			drive-strength = <2>;
+			bias-disable;
+			output-high;
+		};
+	};
+};
+
+&usb3_ss_phy {
+	status = "okay";
+};
+
+&usb3_hs_phy {
+	status = "okay";
+};
+
+&usb2_hs_phy {
+	status = "okay";
+};
+
+&wifi0 {
+	status = "okay";
```

Comment:
Both wifi0 and wifi1 are missing the qcom,ath10k-calibration-variant = "Qxwlan-E2600AC";  property.